examples/fullscreen.rs

```diff
@@ -67,35 +46,14 @@ fn main() {
                 } => match (virtual_code, state) {
                     (VirtualKeyCode::Escape, _) => *control_flow = ControlFlow::Exit,
                     (VirtualKeyCode::F, ElementState::Pressed) => {
-                        #[cfg(target_os = "macos")]
-                        {
-                            if macos_use_simple_fullscreen {
-                                use winit::platform::macos::WindowExtMacOS;
-                                if WindowExtMacOS::set_simple_fullscreen(&window, !is_fullscreen) {
-                                    is_fullscreen = !is_fullscreen;
-                                }
-                                return;
-                            }
-                        }
-
-                        is_fullscreen = !is_fullscreen;
-                        if !is_fullscreen {
+                        if window.fullscreen().is_some() {
                             window.set_fullscreen(None);
                         } else {
-                            window.set_fullscreen(Some(window.current_monitor()));
+                            window.set_fullscreen(fullscreen.clone());
```

Comment:
If we want to use the current monitor here, I think we ought to re-query the video mode from the user as well. I would prefer for this to use whatever values were input by the user on start up, so that we don't have to worry about the monitor changing here.